repository/create_todo_repository.go

```diff
@@ -1,14 +1,15 @@
 package repository
 
 import (
-	"errors"
+	"context"
 	"gorm.io/gorm"
-	"log"
+	"todo-clean/common"
 	"todo-clean/domain"
+	"todo-clean/lib/error_lib"
 	"todo-clean/repository/model"
 )
 
-func (repo *newRepository) CreateTodoRepository(db *gorm.DB, todo domain.CreateTodoEntity) (resp *domain.CreateTodoEntity, err error) {
+func (repo newRepo) CreateTodoRepository(ctx context.Context, db *gorm.DB, todo domain.CreateTodoEntity) (resp *domain.CreateTodoEntity, err error) {
```

Comment:
Note: we can create as list of input entity
e.g. func (repo newRepo) CreateTodoRepository(ctx context.Context, db *gorm.DB, entities []domain.CreateTodoEntity) (resp *domain.CreateTodoEntity, err error)

Reply:
why we should use []

Reply:
In case you want to create many todos you don't have to refactor your code.